### DISCUSSION TITLE
IntelliJ plugin for parsing FIX message using custom dictionary

### DISCUSSION THREAD

**Discussion by alexwibowo** [quickfix-j/quickfixj#880] [Show and tell]
This is not directly related to quickfixj itself, but I've written an IntelliJ plugin that uses quickfixj code to parse FIX message and display them in tree structure.

Here is the plugin: https://plugins.jetbrains.com/plugin/25377-fixtools-fix-protocols-toolset

If you have any idea on how I can improve it, or if you would like to help to work on it, please shoot me email!

(Oh, I also hope I've included all the right license agreement  !)

**Comment by chrjohn** [comment]
Hi @alexwibowo , thanks for lettings us know. Sounds like a nice plugin, although I'm not using IntelliJ myself.
I think it might be sensible to be able to parse also pipe-delimited FIX messages (maybe your plugin does this already, but it is not mentioned).

**Comment by alexwibowo** [comment]
Hi Christoph,

You are right. It already handles pipe delimited message. I should probably
mention it in the description

Best regards,


Alex Wibowo
